### DISCUSSION TITLE
IBM PS/2 Model 30/286 unable to recognize 1.44mb images

### DISCUSSION THREAD

**Discussion by jmahlman** [keirf/flashfloppy#691] [Q&A]
I am able to get the machine to recognize 720k images but will not recognize 1.4MB images. Am using a GoTek Model SFRKC30.AT2 with a Texelec adapter. Current Gotek settings:

Jumpers: J5, S0

FF.CFG:
interface = ibmpc-hdout (have tried ibmpc)
host = unspecified (have tried pc-dos)

These are the only settings I have altered. Any guidance will be helpful.

**Comment by keirf** [comment]
Why jumper j5? Probably not needed. I think I would expect ibmpc-hdout to work. The question is best asked to texelec however.

**Reply by jmahlman** [reply]
Thanks for the reply.  Isn’t J5 needed for High density on the ps2?

wiki states: Also note: You may need to ground pin 4 (place a jumper at J5) for the Gotek to be detected as a High Density drive.

I can try reaching out the texelec…I just find it off that it works with 720k images perfectly fine.

**Reply by keirf** [reply]
Oh yes maybe you're right. Though a regular pc drive connected to this adapter would not ground pin 4.
